handler/error.go

```diff
@@ -5,6 +5,13 @@ type Error struct {
 	logData map[string]interface{}
 }
 
+func NewError(err error, logData map[string]interface{}) *Error {
```

Comment:
function comment before exported function ?
Also, please check with Joshua as to what comments / adjustments he's making to Andre's code that is of a similar nature after the code got reviewed yesterday.